dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest/GetCartIsVirtualTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
```

Comment:
These tests may be excessive since there is no difference in implementation between customer and guest. Even though with such assumption we rely on the details of the implementation, I think it makes sense to avoid unnecessary API functional tests as they are not super-fast to execute.

Alternative solution could be to create alternative PhpUnit config which includes more tests for nightly builds. Will discuss this possibility with @irenedhong who is working on new testing strategy in Magento.

Reply:
@paliarush I have left the Quote/Customer test in there for now as there are a lot of other ones where we already do this. So if the decision is made in the future to consolidate, this test would be one out of 20 to also needing to be consolidated.